server/views/pages/functionalSkills/index.njk

```diff
@@ -33,7 +17,7 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
         <span class="govuk-caption-l">Learning and work progress</span>
-        <h1 class="govuk-heading-l">{{ prisonerSummary.firstName | capitalize }} {{ prisonerSummary.lastName | capitalize }}'s functional skills</h1>
+        <h1 class="govuk-heading-l">{{ prisonerSummary.firstName | trim | capitalize }} {{ prisonerSummary.lastName | trim | capitalize }}'s functional skills</h1>
```

Comment:
Hmm, this is not quite what I had in mind. The problem with this approach is that we're repeating the trim operation all over the place; plus we're making it a view responsibility rather than a "data" type concern. To me it feels like a problem of the data, rather than the view?

I think in the Jira I'd suggested doing something with `PrisonerSummary` - seeing as that is an interface I suspect it would be in `routerRequestHandler#retrievePrisonerSummaryIfNotInSession`

My thinking was if the data is correct in `PrisonerSummary` in the first place we wouldn't need to do all this trimming etc on all the views

What do you think?